src/hooks/sound/useSoundCreatePaymasterEdition.tsx

```diff
@@ -18,6 +28,16 @@ const useSoundCreatePaymasterEdition = () => {
     })
   }
 
+  useEffect(() => {
+    if (callsStatus?.status !== 'CONFIRMED') return
+    toast({
+      title: 'Success',
+      description: 'Published On Sound Successfully!',
+      variant: 'default',
+    })
+    window.location.reload()
+  }, [callsStatus])
```

Comment:
we should already have this code written in `useWaitForTx` or something like that right?